### PR TITLE
some work on the paper

### PR DESCRIPTION
**Pull request by PaulBredl**

### CODE REVIEW

File: paper/design.tex [hunk 1]
```diff
@@ -121,7 +121,7 @@ \subsubsection{Single Source of Truth Data Store}
 % felix
 
 \subsubsection{Code Panel}\label{subsubsec:design_text_editor_panel}
-For the code panel, we embed a rich-text code editor that already supports syntax highlighting and other useful features.
```
Comment (PaulBredl) [inline paper/design.tex:124]: A rich text editor is something like word

File: paper/user_study.tex [hunk 2]
```diff
@@ -23,29 +19,35 @@ \subsection{Research Questions}\label{subsec:research_questions}
...
 \subsection{Methodology}\label{subsec:methodology} % Minye
 
-\paragraph{Potential Users}
+Our user study is qualitative and consists of five interviews.
+We do not perform any major quantitative analysis of the results as the sample size is too small to draw any statistically significant conclusions.
+
+\subsubsection{Potential Users}
 We look for potential users to conduct the interviews.
-There are two basic standards we follow to find a potential user:
+There consider the following groups of people as potential users
```
Comment (PaulBredl) [inline paper/user_study.tex:27]:
```suggestion
We consider the following groups of people as potential users:
```

**Review by beyond2void — DISMISSED** [review]

File: paper/user_study.tex [hunk 3]
```diff
@@ -23,29 +19,35 @@ \subsection{Research Questions}\label{subsec:research_questions}
...
     \item Professors and students who are interested in our application.
-    \item People who frequently use \cfgfiles and schema languages
+    \item People who frequently use \cfgfiles{} and schema languages
 \end{itemize}
+In table~\ref{tab:user_study_feedback}, the professions of the participants are listed.
 
-\paragraph{Interview Questions}
-For the interview, we created a JSON schema and \cfgfile{}, about a made-up self-driving car simulation,
+\subsubsection{Interview Questions}
+We created a JSON schema and \cfgfile{}, about a made-up self-driving car simulation.
```
Comment (PaulBredl) [inline paper/user_study.tex:35]:
```suggestion
We created a JSON schema and \cfgfile{} for a made-up self-driving car simulation software.
```
@beyond2void like that maybe?

Comment (beyond2void) [reply paper/user_study.tex:35]: Oh sorry I mean line 36: "The interview questions deal with working with those files."

Comment (PaulBredl) [reply paper/user_study.tex:35]: True

**Review by beyond2void — DISMISSED** [review]

**Review by beyond2void — APPROVED** [review]
looks good.

**Comment by PaulBredl** [comment]
I would also let @Logende have a look, I modified some of his parts

**Pull request merged**